apps/workflow-server/gcc-workflow-server/src/main/java/com/coremedia/labs/translation/gcc/workflow/GlobalLinkAction.java

```diff
@@ -296,8 +320,12 @@ protected final Result<R> doExecute(Object params) {
     } catch (GlobalLinkWorkflowException e) {
       LOG.warn("{}: " + e.getMessage() + "({})", getName(), e.getErrorCode(), e);
       issues.put(e.getErrorCode(), Collections.emptyList());
+    } catch (RuntimeException e) {
+      // automatically retry upon CMS connection errors
+      return getResultForCMSConnectionError(e);
     }
 
+    result.retryDelaySeconds = gccRetryDelaySeconds;
```

Comment:
I don't understand this assignment. Can you add a comment why retryDelaySeconds needs to be set here?

I might be confused here - but I thought retryDelaySeconds is for automatic retry only, and if we get here there's no automatic retry?

Reply:
There may be issues added by `#doExecuteGlobalLinkAction(...)`. Those would trigger a retry, if we're in a retry loop already, which requires a delay being set. It's been set in the old version, too (line 216).

Added a comment.

Reply:
Comment adjusted to *really* reflect what's going on here.